features/fundraiser_can_create_a_campaign.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Create campaign
+    As an Fundraiser
```

Comment:
Not understanding -- why is this feature file 'create a campaign' in this PR which is for 'User can view campaign details'?  The 'create a campaign' feature was in your previous PR, no?

Reply:
because we had to pull our previous code to have the content we needed to move forward